internal/kyma_test.go

```diff
@@ -17,26 +17,26 @@ func TestRunCmd(t *testing.T) {
 	}{
 		{
 			name:           "Correct command",
-			description:    "Simple test checking that a correct command runs",
+			description:    "Checks if a command is correct.",
 			cmd:            []string{"echo", "Hello!"},
 			expectedOutput: "Hello!\n",
 		},
 		{
 			name:           "Incorrect command",
-			description:    "test a command that exits with error",
-			cmd:            []string{"ehco", "this is wrongly spelled"},
+			description:    "Checks if a command is correct. If not, exists and returns an error.",
+			cmd:            []string{"ehco", "This is spelled incorrectly"},
 			expectedOutput: "",
-			expectedErr:    errors.New("Failed executing command 'ehco [this is wrongly spelled]' with output '' and error message 'exec: \"ehco\": executable file not found in $PATH'"),
+			expectedErr:    errors.New("Executing command 'ehco [this is wrongly spelled]' failed with output '' and error message 'exec: \"ehco\": executable file not found in $PATH'"),
```

Comment:
`"this is wrongly spelled"` should be also replaced in the `expectedErr` field within the `[]`

Reply:
Done.